pkg/cpio/cpio.go

```diff
@@ -84,6 +84,11 @@ func (w Writer) WriteRecord(rec Record) error {
 	return w.rw.WriteRecord(rec)
 }
 
+func (w Writer) WriteReproducibleRecord(rec Record) error {
```

Comment:
Wait a minute. Now that I've been awake for a few hours, it turns I totally agree with you.
Hmm. 
How about just Reproducible, it takes a slice, returns a slice, done?
Damn I missed what you said.